extensions/blocks/story/edit.native.js

```diff
@@ -227,18 +228,30 @@ class StoryEdit extends React.Component {
 		const hasContent = !! mediaFiles.length;
 		const { isUploadInProgress, isSaveInProgress, didUploadFail, didSaveFail } = this.state;
 
+		const mediaPlaceholder = (
+			// TODO this we are wrapping in a pointerEvents=none because we don't want to
+			// trigger the ADD MEDIA bottom sheet just yet, but only give the placedholder the right appearance.
+			<View pointerEvents="none" style={ styles[ 'wp-story-container' ] }>
+				<MediaPlaceholder
+					icon={ <BlockIcon icon={ icon } /> }
+					labels={ {
+						title: __( 'Story' ),
+						instructions: __( 'ADD MEDIA' ),
+					} }
+					allowedTypes={ [ 'image', 'video' ] }
```

Comment:
We can import `MEDIA_TYPE_IMAGE, MEDIA_TYPE_VIDEO` from `core/block-editor` to avoid hard typed strings

Reply:
👍 tackled in deb66b9f38 🙇